Makefile

```diff
@@ -1,11 +1,23 @@
 SOURCE_FILES := $(shell find . -type f -name '*.go')
+VERSION := $(shell git describe --tags --abbrev=0 | cut -c2-)
```

Comment:
```suggestion
VERSION := $(shell git describe | cut -c2-)
```

Reply:
done in https://github.com/kubewarden/hostpaths-psp-policy/pull/14/commits/1fe372d56cf24a2234f517c565170aee49363ae9.